frontend/js/src/settings/Settings.tsx

```diff
@@ -50,13 +63,79 @@ export default function Settings() {
     setShowToken(!showToken);
   };
 
+  const submitFlairPreferences = async () => {
+    if (!currentUser?.auth_token) {
+      toast.error("You must be logged in to update your preferences");
+      return;
+    }
+    const response = await APIService.submitFlairPreferences(
+      currentUser?.auth_token,
+      selectedFlairs
+    );
+    toast.success("Flair preferences updated successfully");
+
+    globalContext.flair = selectedFlairs;
+  };
+
   return (
     <>
       <Helmet>
-        <title>User {currentUser?.name}</title>
+        <title>User {name}</title>
       </Helmet>
       <div id="user-profile">
-        <h2 className="page-title">{name}</h2>
+        <h2 className="page-title">User Settings</h2>
+        <div>
+          <h4>Username: {name}</h4>
+          <a
+            href={`https://musicbrainz.org/user/${name}`}
+            aria-label="Edit Profile on MusicBrainz"
+            title="Edit Profile on MusicBrainz"
+            className="btn btn-outline"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              src="/static/img/meb-icons/MusicBrainz.svg"
+              width="18"
+              height="18"
+              alt="MusicBrainz"
+              style={{ verticalAlign: "bottom" }}
+            />{" "}
+            Edit Profile on MusicBrainz
+          </a>
+        </div>
+
+        <div className="mb-15 donation-flairs-settings">
+          <div className="form-group">
+            <h3>Flair Settings</h3>
+            <p>
+              Choose which flairs do you want to use for donation in
```

Comment:
Maybe something like this ?
```suggestion
              Choose which flair you want your username to show to let other users know you donated.
```